Shopify/cli
# Update ink to 4.4.1

### WHY are these changes introduced?

4.4.1 introduces a fix that we need for Windows and the `Tasks` component `useInput` usage.

### WHAT is this pull request doing?

Update ink to 4.4.1

### How to test your changes?

Run all the kitchen sink commands and see that they work correctly.

- `shopify kitchen-sink prompts`
- `shopify kitchen-sink async`
- `shopify kitchen-sink static`

- Run `dev` on a new app, see that the footer includes shortcuts.
- Run `pnpm shopify app generate extension` on a new app in Windows. It should work until the end.
